### PR TITLE
etcdctlv3: make del command better

### PR DESCRIPTION
**Pull request by xiang90**
Also rename delete-range to del, which is shorter and better (since the range is optional)

### CODE REVIEW

File: etcdctlv3/command/del_command.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	}
+
+	c := mustClientFromCmd(cmd)
+	kvapi := clientv3.NewKV(c)
+	_, err := kvapi.Delete(context.TODO(), key, opts...)
+	if err != nil {
+		ExitWithError(ExitError, err)
+	}
+	// TODO: add number of key removed into the response of delete.
+	// TODO: print out the number of removed keys.
```
Comment (gyuho) [inline etcdctlv3/command/del_command.go]: Q. how are we going to do this?

Comment (xiang90) [inline etcdctlv3/command/del_command.go]: the server will return the number of removed keys. the client side just needs to print it out.

Comment (gyuho) [inline etcdctlv3/command/del_command.go]: I see. Thanks!

File: etcdctlv3/README.md [hunk 2]
```diff
@@ -42,3 +42,31 @@ Insert '--' for workaround:
 ./etcdctl put -- <key> <value>
 ```
 
+### DEL [options] \<key\> [range_end]
+
+Removes the specified key or range of keys [key, range_end) if `range-end` is given. A key is ignored if it does not exist.
```
Comment (heyitsanthony) [inline etcdctlv3/README.md]: I don't think "A key is ignored if it does not exist." is necessary, it's a little confusing.

Comment (xiang90) [inline etcdctlv3/README.md]: ok. will remove.

File: etcdctlv3/README.md [hunk 3]
```diff
@@ -42,3 +42,31 @@ Insert '--' for workaround:
...
+
+#### Options
+
+None.
```
Comment (heyitsanthony) [inline etcdctlv3/README.md]: `--prefix`? `--from`?

Comment (xiang90) [inline etcdctlv3/README.md]: will add as todo.

**Comment by xiang90** [comment]
@heyitsanthony PTAL

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**